google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractMultiplexedSessionDatabaseClient.java

```diff
@@ -56,4 +57,10 @@ public ServerStream<BatchWriteResponse> batchWriteAtLeastOnce(
   public long executePartitionedUpdate(Statement stmt, UpdateOption... options) {
     throw new UnsupportedOperationException();
   }
+
+  @Override
+  public long executePartitionedUpdate(
+      Statement stmt, @Nullable String transactionTag, UpdateOption... options) {
```

Comment:
Should we create TransactionOption instead of String so that it can be extendable in future?

Reply:
For all other options we already have `UpdateOption`, only reason we are adding transaction tag here because it collides with TagOption in UpdateOption otherwise we would have used UpdateOption iteself.